ui/changes/src/index.js

```diff
@@ -523,6 +557,16 @@ async function buildTable(rerender = true) {
       bugSummary["whiteboard"].includes(whiteBoard)
     );
   }
+  if (releaseVersion) {
+    bugSummaries = bugSummaries.filter((bugSummary) => {
+      for (let item of bugSummary.versions) {
+        for(let version of releaseVersion) {
+          if (item == version)
+            return bugSummary
+        }
+      }
```

Comment:
You could use `some` and `includes` to make this super readable:
```suggestion
      return bugSummary.versions.some(version => releaseVersions.includes(version));
```

This would mean: at least one version of the bug is in the selected versions.

Reply:
I could only get this to work like this: 
```releaseVersion.some(version => bugSummary.versions.some(item => item == version)));```

Reply:
Mmmh, maybe it's because `bugSummary.versions` is a list of ints, while `releaseVersions` is a list of strings.
You could try:
```
return bugSummary.versions.some(version => releaseVersions.includes(version.toString()));
```
or
```
return releaseVersions.some(version => bugSummary.versions.includes(Number(version)));
```